README.md

```diff
@@ -1,8 +1,117 @@
-# CircularEconomy-MassFlowCalculator
+<img src="docs/images_wiki/CE-MFC.png" width="400">
+
+<table>
+<tr>
+  <td>License</td>
+  <td>
+    <a href="https://github.com/NREL/CircularEconomy-MassFlowCalculator/blob/master/LICENSE">
+    <img src="https://img.shields.io/pypi/l/pvlib.svg" alt="license" />
+    </a>
+</td>
+</tr>
+<tr>
+  <td>Build Status</td>
+  <td>
+    <a href="http://circulareconomy-massflowcalculator.readthedocs.org/en/latest/">
+    <img src="https://readthedocs.org/projects/circulareconomy-massflowcalculator/badge/?version=latest" alt="documentation build status" />
+    </a>
+  </td>
+</tr>
+<tr>
+  <td>Publications</td>
+  <td>
+    <!--- <a href="https://doi.org/10.5281/zenodo.3762635">
+    <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.3762635.svg" alt="zenodo reference">
+     ---> Connected to Zenodo, awaiting first release
+    </a>
+  </td>
+</tr>
+</table>
+
+
+# CEMFC: Circular Economy - Mass Flow Calculator
 
 This open-source tool will help quantify and assign a value framework to efforts on re-design, reduction, replacement, reusage, recycling, and lifetime and reliability increases on PV, implementing Circular Economy metrics for PV materials.
```

Comment:
```suggestion
This open-source tool implements Circular Economy metrics for photovoltaic (PV) materials. It can be used to quantify and assign a value framework to efforts on re-design, reduction, replacement, reuse, recycling, and lifetime and reliability increases in the PV value chain.
```